app/javascript/components/enrollment/steps/Review.js

```diff
@@ -117,7 +117,7 @@ Review.propTypes = {
   previous: PropTypes.func,
   goto: PropTypes.func,
   submit: PropTypes.func,
-  parent_id: PropTypes.string,
+  parent_id: PropTypes.number,
```

Comment:
Good catch - though it looks like this is never even used in this component.

Reply:
good catch I will just remove it

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/737/commits/8e671f47720b3de9006b03fbc0ba2d96d4a9668f